Fails to install if path contains spaces

To reproduce, create a crystal project with a space in its path and try to install ameba in it.
```
~/tmp/space path/testapp> shards install
Fetching https://github.com/crystal-ameba/ameba.git
Installing ameba (0.10.0)
Postinstall make bin
Failed make bin:
/usr/bin/shards build 
Dependencies are satisfied
Building: ameba
Error target ameba failed to compile:
Showing last frame. Use --error-trace for full trace.

In src/ameba.cr:28:16

 28 | VERSION = {{ `shards version #{__DIR__}`.chomp.stringify }}
                   ^
Error: error executing command: shards version /home/cassidy/tmp/space path/testapp/lib/ameba/src, got exit status 1:

Missing space. Please run 'shards init'


make: *** [Makefile:8: bin/ameba] Error 1
```

--- a/src/ameba.cr
+++ b/src/ameba.cr
@@ -24,7 +24,7 @@ require "./ameba/formatter/*"
 module Ameba
   extend self
 
-  VERSION = {{ `shards version #{__DIR__}`.chomp.stringify }}
+  VERSION = {{ `shards version "#{__DIR__}"`.chomp.stringify }}
 
   # Initializes `Ameba::Runner` and runs it.
   # Can be configured via `config` parameter.

